.github/renovate.json

```diff
@@ -4,26 +4,12 @@
   "extends": ["config:recommended"],
   "packageRules": [
     {
-      "groupName": "github actions",
-      "matchManagers": ["github-actions"],
-      "matchUpdateTypes": ["major", "minor"],
-      "extends": ["helpers:pinGitHubActionDigests"],
-      "minimumReleaseAge": "14 days"
-    },
-    {
-      "groupName": "npm dependencies",
+      "groupName": "dependencies for ui-components",
+      "matchFileNames": ["packages/ui-components/package.json"],
```

Comment:
Yes that is possible but then there will be a lot of PRs generated all at once because deps are quite outdated therefore doing it step by step for each package would make less PRs opened simultaneously. See my other comment.

Reply:
Ahh ok! I got it.